sysl2/sysl/seqs.go

```diff
@@ -179,7 +186,7 @@ func DoGenerateSequenceDiagrams(stdout, stderr io.Writer, args []string) error {
 			"comma-list of shell globs) to limit the diagrams generated",
 	).Short('a').Strings()
 
-	blackboxesFlag := sd.Flag("blackbox", "Apps to be treated as black boxes").Strings()
+	blackboxesFlag := sd.Flag("blackbox", "Apps to be treated as black boxes").Short('b').StringMap()
```

Comment:
Add a bit about blackbox should be in the following format: `App <- Endpoint=Some description`
And that you can pass multiple `-b` args.

Reply:
and add a test case where you pass  incorrect format i.e. missing `=Comment`